examples/ml/ml-cronjob.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: ml.arangodb.com/v1alpha1
```

Comment:
Remove example, we will provide example in docs later

Reply:
why? it is useful for quick tests...

Reply:
We will generate it in examples and readme for all objects